cl/favorites/tasks.py

```diff
@@ -31,9 +31,16 @@ def check_prayer_pacer(self, rd_pk: int, user_pk: int):
     """
     rd = (
         RECAPDocument.objects.select_related("docket_entry__docket")
-        .defer("plain_text")
+        .only(
+            "pacer_doc_id",
+            "docket_entry_id",
+            "docket_entry__docket_id",
+            "docket_entry__docket__court_id",
+            "is_sealed",
+        )
```

Comment:
Is there a reason to only select those fields in the query?

If we end up calling `prayer_unavailable` to send the email, it'll actually cause more database queries down the line. That's because the email needs things like the `docket number`, uses the `best_case_name` function, and pulls other info from the docket entry and recap document models.